vendor/lowrisc_ip/ip/prim/lint/prim_arbiter.vlt

```diff
@@ -39,6 +39,8 @@ lint_off -rule ALWCOMBORDER -file "*/rtl/prim_arbiter_tree.sv" -match "*data_tre
 
 lint_off -rule ALWCOMBORDER -file "*/rtl/prim_arbiter_ppc.sv" -match "*ppc_out*"
 
+lint_off -rule UNOPTFLAT -file "*/rtl/prim_arbiter_ppc.sv" -match "*winner*"
```

Comment:
This should be put in a patch file I think.

Reply:
Yes, that was the intent, thanks. Take two.